test-lit/lit.site.cfg.py.in

```diff
@@ -3,8 +3,6 @@
 import sys
 
 config.llvm_tools_dir = "@LLVM_TOOLS_DIR@"
```

Comment:
Ah - now I think I understand the LLVM_TOOLS_BINARY_DIR need. I think you have used LLVM_TOOLS_BINARY_DIR and LLVM_TOOLS_DIR somewhat interchangeably in this patchset but, unless I am misunderstanding things, they mean to be the same thing. I think if you make it all line up this should work.

Reply:
Done!